lms/djangoapps/student_account/views.py

```diff
@@ -133,9 +133,7 @@ def login_and_registration_form(request, initial_mode="login"):
             'third_party_auth_hint': third_party_auth_hint or '',
             'platform_name': configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME),
             'support_link': configuration_helpers.get_value('SUPPORT_SITE_LINK', settings.SUPPORT_SITE_LINK),
-            'password_reset_support_link': configuration_helpers.get_value(
-                'PASSWORD_RESET_SUPPORT_LINK', settings.SUPPORT_SITE_LINK  # Intentional default.
-            ),
```

Comment:
@bradenmacdonald I had tried that and it indeed worked, but was worried about the default. If it just takes it from `aws.py`, then OK, no problem there then about the default. Going to test if it defaults to `SUPPORT_SITE_LINK` if we don't do the 3rd version there.